hieradata/clients/controller.sponsorship.ci.jenkins.io.yaml

```
@@ -392,6 +392,44 @@ profile::jenkinscontroller::jcasc:
           credentialsId: "jenkinsvmagents-userpass"
           usePrivateIP: true
           spot: true
+        - name: "ubuntu-22-inbound-amd64-jdk17"
```

```suggestion
        - name: "ubuntu-22-amd64-maven17"
```

- Remove the `inbound` element as it is an implementation detail (and can be changed in the template). It does not give any useful information to the end user (only to administrators) so better removing it
- Using the same naming as the existing container agent: `maven17` instead of `jdk17` (ref. https://github.com/jenkinsci/jenkins/blob/0d5232fca40a12b2d14f7ded577a90f0cb3c6f06/Jenkinsfile#L22 as example of consumption)